README.md

```diff
@@ -108,6 +108,32 @@ module.exports = {
 }
 ```
 
+### Disabling HMR
+
+By default Hot Module Replacement is disabled in following situations:
+
+ * Webpack `target` is `node`
+ * Webpack doesn't minify the code 
+ * `process.env.NODE_ENV !== 'production'`
```

Comment:
* Webpack `target` is `node`
* Webpack minifies the code
* `process.env.NODE_ENV === 'production'`

is correct 🙂